keps/sig-api-machinery/20190425-crd-conversion-webhook.md

```diff
@@ -736,8 +735,23 @@ Consider including folks that also work outside the SIG or subproject.
 
 ## Graduation Criteria
 
+v1beta1:
```

Comment:
before graduating to beta, we must finalize validation restrictions on what metadata fields conversion is required to preserve (xref https://github.com/kubernetes/kubernetes/issues/72160)

Reply:
Added.